cvmfs/swissknife_pull.cc

```diff
@@ -571,6 +569,10 @@ int swissknife::CommandPull::Main(const swissknife::ArgumentList &args) {
   download_manager()->SetRetryParameters(retries, timeout, 3*timeout);
   download_manager()->Spawn();
 
+  // initializing the threads now in case of cleanup
```

Comment:
Depending on when the threads are initialized they might have to be deleted (cleaned up). The idea, as it was mentioned before, is to delay as much as possible the creation of the threads so that it avoids these points (error cases mostly) where the function exits, because otherwise it'd be mandatory to put the corresponding _delete[] workers_. That being said, there is no special reason for this comment and it could be perfectly deleted.